test/parallel/test-worker-stack-overflow.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const { Worker } = require('worker_threads');
+
+const worker = new Worker('function f() { f(); } f();', { eval: true });
+
+worker.on('error', common.mustCall((err) => {
+  // assert.strictEqual(err.constructor, RangeError);
```

Comment:
🙈

Reply:
@cjihrig Done! This part of the test was actually written before https://github.com/nodejs/node/pull/25951 :)

Reply:
The fact that it's commented out means it isn't actually a RangeError?